src/test/java/edu/harvard/iq/dataverse/DatasetVersionTest.java

```diff
@@ -121,7 +124,7 @@ public void testGetJsonLd() throws ParseException {
         assertEquals("https://doi.org/10.5072/FK2/LK0D1H", obj.getString("identifier"));
         assertEquals(null, obj.getString("schemaVersion", null));
         assertEquals("Dataset", obj.getJsonObject("license").getString("@type"));
-        assertEquals("CC0", obj.getJsonObject("license").getString("text"));
+        assertEquals("CC0", obj.getJsonObject("license").getString("name"));
```

Comment:
Hmm - this is actually what provides the license metadata to Google and Google complains that we should be sending something else. 
So - 'text' is bad, 'name' is slightly better (just tested on google and if we sent a"name" instead of "text' and our @type were CreativeWork instead of Dataset it works), but, since we now have a URL for both standard and custom licenses, it looks like we can just send `"license":"https://creativecommons.org/publicdomain/zero/1.0/" ` instead. (Or a license object with @type CreativeWork and a "name" and "url" - the single value seems easier.)

TLDR: Looks like this is a place where we can probably fix an existing problem because the multilicense PR introduces URLs for all standard/custom licenses. Which won't be backward compatible, but will be better.